packages/experimental/Badge/src/CounterBadge/useCounterBadge.ts

```diff
@@ -0,0 +1,21 @@
+import { CounterBadgeProps, CounterBadgeInfo } from './CounterBadge.types';
+
+export const useCounterBadge = (props: CounterBadgeProps): CounterBadgeInfo => {
+  const DEFAULT_OVERFLOW_COUNT = 99;
+  const { dot, overflowCount = DEFAULT_OVERFLOW_COUNT, showZero, count, ...rest } = props;
+
+  const _count = count && count > overflowCount ? `${overflowCount}+` : `${count}`;
+  const showBadge = count !== 0 || (count === 0 && !!showZero);
```

Comment:
WHat's expected to happen if the passed in count is negative?

Reply:
The negative number will be shown. I checked how it works for Web badge